xml/obs_moderation.xml

```diff
@@ -0,0 +1,297 @@
+          <imageobject>
+          <imagedata fileref="report_user.png" width="800"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+      <para>
+        When reporting a comment, the form allows you to report the comment's author as well.
+        Which saves you from having to go to the user's profile page afterwards.
+      </para>
+
+      <figure>
+        <title>Report Comment and Author</title>
+        <mediaobject>
+          <imageobject>
+          <imagedata fileref="comment_report.png" width="800"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+      <para>
+        Once the element is reported, moderators will act accordingly.
+        The following sections describe how they should proceed.
+      </para>
+    </sect2>
+  </sect1>
+
+  <sect1>
+    <title>Acting as a Moderator</title>
+    <para>
+      Moderators should inspect the elements reported by the users
+      to decide whether the reports are fair or not, and act accordingly.
+      They can dismiss the report, hide comments or remove the problematic elements, including users.
+    </para>
+    <sect2>
+      <title>Who Is a Moderator?</title>
+      <para>
+        Admins and staff members are the default moderators in any OBS instance unless they delegate the role to another user.
+        Going through <emphasis>Configuration > Manage Users</emphasis>, admins can assign the <emphasis>Moderator</emphasis> role to someone else.
+        From that moment on, only the users with the <emphasis>Moderator</emphasis> role will be involved in the moderation process.
+      </para>
+    </sect2>
+
+    <sect2>
+      <title>How Do Moderators Know About the Reports?</title>
+      <para>
+        Next to any problematic element, they can see a yellow text warning about the reports.
+      </para>
+
+      <figure>
+        <title>Reports Warning</title>
+        <mediaobject>
+          <imageobject>
+          <imagedata fileref="reports_warning.png" width="800"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+        <para>
+        However, it is convenient that moderators subscribe to moderation-related events,
+        so they can receive notifications of all the reports and other actions related to the moderation process.
+      </para>
+
+      <figure>
+        <title>Moderation-related Notifications</title>
+        <mediaobject>
+          <imageobject>
+          <imagedata fileref="moderation-related_notifications.png" width="800"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+    </sect2>
+
+    <sect2>
+      <title>How To Moderate?</title>
+      <para>
+        As a moderator you can click on the yellow warning about reports, usually displayed close to the element.
+        There, you can read all the reports and make a <emphasis>decision</emphasis>. Write the reason why you agree, <emphasis>Favor</emphasis>, or disagree, <emphasis>Cleared</emphasis>.
```

Comment:
```suggestion
        There, you can read all the reports and make a <emphasis>Decision</emphasis>. Write the reason why you agree, <emphasis>Favor</emphasis>, or disagree, <emphasis>Cleared</emphasis>.
```
Also, decision should be capitalized, like we do with Favor and Cleared, because those are concepts we use across the UI (part of our "shared language", like talking about Packages and Projects or BsRequests).

Reply:
As an alternative you could also create a glossary entry and link to it. That would make it crystal clear and is independent of any spelling.

Reply:
@tomschr Done!
Could you please have a look at the new glossary entry and the cross-reference, please?

Reply:
@saraycp Looks good! :+1: Thanks for adding it (I just thought, it was a crazy idea. :laughing:)

Reply:
It's a good idea and we could probably add more words related to the most recent features. This is a good start!